common/src/gas_provider/node.rs

```diff
@@ -37,7 +63,13 @@ pub enum GasNode<ExternalId: Clone, Id: Clone, Balance: Zero + Clone> {
     ///
     /// Such node types are independent and aren't part of the tree structure
     /// (not node's parent, not node's child).
-    ReservedLocal { id: ExternalId, value: Balance },
+    Cut { id: ExternalId, value: Balance },
+
+    /// A node used for gas reservation feature.
+    ///
+    /// Such node types are independent and aren't part of the tree structure
+    /// (not node's parent, not node's child).
+    Reserved { id: ExternalId, value: Balance },
```

Comment:
Probably, reserved should also contain "parent" key for future purposes

Reply:
So why not leave it "as is" for now?